incubator/monochart/templates/statefulset.yaml

```
@@ -61,7 +61,7 @@ spec:
           protocol: {{ default "TCP" $port.protocol  }}
 {{- end }}
         volumeMounts:
-        - mountPath: {{ .Values.persistence.mountPath | quote }}
+        - mountPath: {{ default "/data" (default .Values.persistence.mountPath .Values.statefulset.persistence.mountPath) | quote }}
```

No reason in `default "/data"` because https://github.com/cloudposse/charts/pull/187/files#diff-bde951bc4f092e56d2f2b85222c85c31R317

```suggestion
        - mountPath: {{ default .Values.persistence.mountPath .Values.statefulset.persistence.mountPath | quote }}
```

---

Why `.Values.statefulset.persistence.mountPath` have special `mountPath` ?